presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -361,3 +361,20 @@ can be reused.  For example, one may be interested in a daily reading of the 99t
 percentile values that are read over the course of a week.  Instead of calculating
 the past week of data with ``approx_percentile``, ``qdigest``\ s could be stored
 daily, and quickly merged to retrieve the 99th percentile value.
+
+T-Digest
+---------------
+
+.. _tdigest_type:
+
+``TDigest``
+^^^^^^^^^^^
+
+A t-digest (tdigest) is a summary structure which, similarly to qdigest, captures the
+approximate distribution of data for a given input set, and can be queried to retrieve
+approximate quantile values from the distribution.  The main advantage of tdigest over
```

Comment:
Also performance and memory utilization. T-digest is more efficient to operate on that qdigest.